### ISSUE TITLE
NVDA not announcing aria-label on lists

### ISSUE THREAD

**Issue by Venkatesha11y** [nvaccess/nvda#7652]
Steps to reproduce:

1. Go to https://codepen.io/Venkatesh-a11y/pen/OxZWdv
2. As soon as page loads focus lands on edit field so hit ESC key to switch to browse mode.
3. Hit M to move to the result frame.
4. Now arrow down to both unordered and ordered list over there.
5. Or hit L to navigate lists on the page.

Expected behavior:

NVDA should announce the aria-label before announcing the list item when focus enters list with quick key L or with down arrow otherwise context will miss for the users though specific context is described for each and every list with aria-label.
In the mentioned example, when user hits L NVDA should announce "list with two items here are the list of fruits I like bullet apples" and in the next hit of L it should announce "list with two items here are the list of vegetables I like one potato".

Actual Result:

NVDA doesn't announce the aria-label with quick key L or while arrowing down the list instead it announces the number of list items followed by the first list item directly.
In the mentioned example, NVDA announces "list with two items bullet Apples" and "list with two items one Potato" when focus enters lists.

System configuration:

NVDA Version: 2017.3, Installed

Other information:

OS version: Windows 10
Browser Versions: Firefox 56, Chrome 61, IE 11, Microsoft Edge 41
Tested with NVDA 2017.1 and 2017.2 also and the issue repro's.

**Comment by feerrenrut** [comment]
I can reproduce this, and I agree this does not seem like ideal behaviour. I have a feeling I have seen a similar issue raised previously, the best I could find right now are:
* Report the Title Attribute on HTML lists and Tables #2137
* Report label in browse mode if it isn't rendered anywhere else (e.g. aria-label) #4773

Is there a past version of NVDA where this used to work as expected?

**Comment by yonyy** [comment]
I am still seeing this issue with NVDA version 2018.2.1

**Comment by Adriani90** [comment]
cc: @michaelDCurran
